internal/characters/nahida/attack.go

```diff
@@ -75,12 +75,20 @@ func (c *char) Attack(p map[string]int) action.ActionInfo {
 		attackHitmarks[c.NormalCounter],
 	)
 
+	cancel := attackHitmarks[c.NormalCounter]
```

Comment:
I'm dumb, we could've just set the `CanQueueAfter` to the swap action, since it's the fastest cancel for every normal attack. I'll leave this up to you if you want to change it.